src/cards_management.c

```diff
@@ -340,7 +340,7 @@ Deck_t *remove_first_playable_card(Deck_t **pp_head)
                 prev->next = cur->next; // Adjust links for previous node
             }
             //free(cur);                  // Delete current node
-            printf("Successfully deleted the first playable card (%s, %s) in the middle. \n", CARD_COLOR_STRING[cur->card.color], CARD_NAME_STRING[cur->card.name]);
+            //printf("Successfully deleted the first palyable card (%s, %s) in the middle. \n", CARD_COLOR_STRING[cur->card.color], CARD_NAME_STRING[cur->card.name]);
```

Comment:
Let's delete the above all the comment out lines.

Reply:
We'll delete if after release 2.